src/Corax/Utils/CoraxSpatialPointEntry.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using Sparrow;
+
+namespace Corax.Utils;
+
+public readonly struct CoraxSpatialPointEntry
+{
+    public readonly double Latitude;
+
+    public readonly double Longitude;
+
+    public readonly string Geohash;
+
+    public CoraxSpatialPointEntry(double latitude, double longitude, string geohash)
+    {
+        Latitude = latitude;
+        Longitude = longitude;
+        Geohash = geohash;
+    }
+
+    public Span<byte> GeohashAsBytes
+    {
+        get
+        {
+            return Encodings.Utf8.GetBytes(Geohash);
```

Comment:
Is there any way we can get away without passing strings around?

Reply:
@maciejaszyk ping